README.rst

```diff
@@ -1,14 +1,12 @@
-
-README for Pylint - https://pylint.pycqa.org/
-=============================================
+Pylint
```

Comment:
Can't we just use the README as `index.html`? I did the same for `pydocstringformatter`. We wouldn't need the duplication then.

Reply:
Done